scheduler/cloudformation/cloudformation.go

```diff
@@ -941,3 +1039,61 @@ func (e *templateValidationError) Error() string {
   Error: %v`
 	return fmt.Sprintf(t, e.templateURL, e.templateBody.Len(), e.err)
 }
+
+// output returns the cloudformation.Output that matches the given key.
+func output(stack *cloudformation.Stack, key string) (output *cloudformation.Output) {
+	for _, o := range stack.Outputs {
+		if *o.OutputKey == key {
+			output = o
+		}
+	}
+	return
+}
+
+type ecsDeployment struct {
+	service    string
```

Comment:
I would change this to `process` or `processName`, so `service` doesn't get confused with the ECS service.